src/mqt/ionshuttler/multi_shuttler/outside/graph.py

```
@@ -35,7 +36,11 @@ class Graph(nx.Graph):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)